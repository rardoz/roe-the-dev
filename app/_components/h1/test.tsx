import H1 from '.'
import { render } from '@testing-library/react'

describe('H1', () => {
  it('should render', () => {
    expect(render(<H1>TEST</H1>).container).toMatchSnapshot()
  })
  it('should have title set to Test 123', () => {
    expect(render(<H1>Test 123</H1>).container.textContent).toBe('Test 123')
  })
})
