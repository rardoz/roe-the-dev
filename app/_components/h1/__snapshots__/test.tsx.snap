// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 should render 1`] = `
<div>
  <h1
    class="mb-4 text-4xl font-extrabold leading-none tracking-tight text-gray-900 md:text-5xl lg:text-6xl dark:text-white"
  >
    TEST
  </h1>
</div>
`;
